src/svm.cpp

```diff
@@ -133,11 +133,8 @@ NAN_METHOD(enqueueSVMFree) {
     Local<Array> js_events = Local<Array>::Cast(info[4]);
     NOCL_TO_ARRAY(cl_events, js_events, NoCLEvent);
   }
-  cl_event* eventPtr = nullptr;
   cl_event event;
-
-  if(ARG_EXISTS(5) && info[5]->BooleanValue())
-      eventPtr = &event;
+  cl_event* eventPtr = &event;
```

Comment:
unsafe here + after?

Reply:
It should be safe, I was trying not to leave anything unsafe during implementation, off top of my head I didn't like the polymorphic return type, no developer likes surprises. Also `CHECK_ERR(err);` seems to throw an error upon non-null error, so the `if` clause is pointless its negative branch will never be executed so why bother creating a conditional return type?

Reply:
It's safe  CHECK_ERR(err) throws exception so `info.GetReturnValue().Set(JS_INT(CL_SUCCESS));` will never be executed